code/modules/gear_presets/uscm_medical.dm

```diff
@@ -44,34 +44,20 @@
 	minimap_background = MINIMAP_ICON_BACKGROUND_CIC
 
 	utility_under = list(/obj/item/clothing/under/rank/chief_medical_officer)
-	utility_hat = list(/obj/item/clothing/head/cmo)
-	utility_gloves = list(/obj/item/clothing/gloves/latex)
+	utility_hat = list()
+	utility_gloves = list()
 	utility_shoes = list(/obj/item/clothing/shoes/white)
-	utility_extra = list(/obj/item/clothing/suit/storage/labcoat)
+	utility_extra = list()
 
 /datum/equipment_preset/uscm_ship/uscm_medical/cmo/load_gear(mob/living/carbon/human/new_human)
-	var/back_item = /obj/item/storage/backpack/marine/satchel
+	var/back_item = /obj/item/storage/backpack/satchel
 	if (new_human.client && new_human.client.prefs && (new_human.client.prefs.backbag == 1))
 		back_item = /obj/item/storage/backpack/marine
 
-	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/cmo(new_human), WEAR_L_EAR)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/rank/chief_medical_officer(new_human), WEAR_BODY)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/white(new_human), WEAR_FEET)
-	new_human.equip_to_slot_or_del(new /obj/item/clothing/gloves/latex(new_human), WEAR_HANDS)
-	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/labcoat(new_human), WEAR_JACKET)
-	new_human.equip_to_slot_or_del(new /obj/item/paper/research_notes/decent(new_human), WEAR_IN_JACKET)
-	if(new_human.disabilities & NEARSIGHTED)
-		new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/hud/health/prescription(new_human), WEAR_EYES)
-	else
-		new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/hud/health(new_human), WEAR_EYES)
```

Comment:
Why remove this?

Reply:
Yes...I removed it and added to the vendors, yes, but Fira complained about it and, because of the new PR that gives the possibility of adding prescription lens to them from the glasses they will wake up with, they asked to remove it

Reply:
You are giving a conflicting request